spec/forms/openid_connect_authorize_form_spec.rb

```diff
@@ -357,7 +357,7 @@
   end
 
   describe '#aal' do
```

Comment:
This block is mostly testing `#requested_aal_value` now, but I think it would be valuable to keep the current `#aal` tests mostly as-is, and add tests for `#requested_aal_value` in a separate `describe '#requested_aal_value' do` block.

Reply:
Reverted the AAL tests, and added the Requested AAL Value tests in addition.

Reply:
Looks like its the AAL1 tests that are failing, which makes sense since we don't support them now based on [this document](https://docs.google.com/document/d/1XP-HdB7yd9XuxMoB4CmGisvaYWClY1g0cZXwmBTRA_Y/edit#) outlining the priority order of AALs.